src/hooks/utils/useGetAccountName.ts

```
@@ -16,10 +17,12 @@ export const createAccountSubstring = (account: string) => {
  * @todo Should switch to object for props
  */
 export const useGetAccountName = (account?: Address | null, truncate?: boolean) => {
+  const { chain } = useNetworkConfigStore();
   if (truncate === undefined) truncate = true;
 
   const { data: ensName } = useEnsName({
     address: !!account ? account : undefined,
+    chainId: chain.id,
```

But here though, I think this is a place to use mainnet specifically for this call, not the chain from useNetworkConfigStore.

we're doing reverse lookups here, so we're trying to get human readable names. just display purposes only.

i think it's safe to leave this change as is, or hardcode mainnet. but i like leaving it as it is.

---

On some chains ENS is supported separately - like Sepolia / OP
So I'd let it fail under the hood instead of always displaying ENS from mainnet

---

This is what I was thinking about. maybe this should be a seperate issue. as we do have hardcoded mainnet is other places. maybe we should take a second and reconcile these different places.

---

Yeah good call @Da-Colon . Although is this change here necessary for this PR? If not, can you remove it from this PR and create an issue?

---

yeah the changes here keep things as they were in the app. I think its safe to leave this and update it from here.

---

Agree with @mudrila's take here

---

Issue created https://linear.app/decent-labs/issue/ENG-137/ens-updates